modules/nodes-sigstore-configure-image-policy.adoc

```diff
@@ -0,0 +1,203 @@
+    - example.com
+  policy: <4>
+    rootOfTrust: <5>
+      policyType: FulcioCAWithReko <6>
+      publicKey: <7>
+        fulcioCAData: a2V5RGF0YQ==
+        fulcioSubject:
+          oidcIssuer: "\https://expected.OIDC.issuer/"
+          signedEmail: "\expected-signing-user\@example.com"
+        rekorKeyData: cmVrb3JLZXlEYXRh <8>
+    signedIdentity:
+      matchPolicy: exactRepository <9>
+      exactRepository:
+        repository: example.com/repository/image <10>  
+----
+<1> Creates an `ImagePolicy` object.
+<2> Specifies the namespace where the image policy is applied. 
+<3> Defines a list of image scopes assigned to this policy.
+<4> Specifies the parameters that define how the images are verified.
+<5> Defines a root of trust for the policy.
+<6> Specifies the policy types that define the root of trust. Here, a Fulcio certification with required Rekor verification.
+<7> For a Fulcio certificate policy, the following parameters are required:
+* `fulcioCAData`: Specifies a base64-encoded Fulcio certificate in the PEM format. The maximum length is 8192 characters.
+* `fulcioSubject`: Specifies the OIDC issuer and the email of the Fulcio authentication configuration.
+<8> Specifies a base64-encoded Rekor public key in the PEM format. This parameter is required when when the `policyType` is `FulcioCAWithReko`. The maximum length is 8192 characters.
+<9> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `matchRepoDigestOrExact` 
+* `matchRepository`.
+* `exactRepository`
+* `remapIdentity`
+<10> For the `exactRepository` match policy, specifies the repository that contains the image identity and signature.
+--
+
+. Create the cluster image policy object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+Scheduling is disabled on each node as it is updated.
+
+.Verification
+
+. After the nodes in your cluster are updated, you can verify that the image policy has been configured:
+
+.. Enter into the node in debug mode by running the following command::
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+.. Set `/host` as the root directory within the debug shell by running the following command:
++
+[source,terminal]
+----
+sh-5.1# chroot /host/
+----
+
+.. Examine the `<namespace>.json` file  by running the following command: 
++
+.Example output
+[source,json]
+----
+sh-5.1# cat /etc/crio/policies/mynamespace.json
+----
++
+.Example output
+[source,json]
+----
+# ...
+  "docker": {
+   "example.com": [
+    {
+     "type": "sigstoreSigned",
+     "keyData": "a2V5RGF0YQ==",
+     "rekorPublicKeyData": "cmVrb3JLZXlEYXRh",
+     "signedIdentity": {
+      "type": "matchRepoDigestOrExact"
+     }
+# ...
+----
+
+////
+..  Check the crio log for sigstore signature verification for namespaced policy by running the following command:
++
+[source,terminal]
+----
+sh-5.1#  journalctl -u crio | grep -A 100 "Pulling image: docker.io"
```

Comment:
```suggestion
sh-5.1#  journalctl -u crio | grep -A 100 "Pulling image: example.io/crio"
```

Reply:
@QiWang19 I was not able to get this command to work for the Fulcio image policy example. Perhaps @lyman9966 Can help verify?